core/src/test/java/org/elasticsearch/search/indicesboost/SimpleIndicesBoostSearchIT.java

```diff
@@ -77,31 +75,154 @@ public void testIndicesBoost() throws Exception {
         logger.info("Hit[1] {} Explanation {}", response.getHits().getAt(1).index(), response.getHits().getAt(1).explanation());
         assertThat(response.getHits().getAt(0).index(), equalTo("test2"));
         assertThat(response.getHits().getAt(1).index(), equalTo("test1"));
+    }
 
-        logger.info("--- DFS_QUERY_THEN_FETCH");
+    public void testIndicesBoostWithAlias() throws Exception {
+        createIndex("alias_test1", "alias_test2");
+        ensureGreen();
+        client().admin().indices().prepareAliases().addAlias("alias_test1", "alias1").get();
+        client().admin().indices().prepareAliases().addAlias("alias_test2", "alias2").get();
```

Comment:
you can add the aliases as part of the index creation, that may make the test a bit faster ;)